docs/user-guide/administration/manage-metadata/metadata-storage/configuration.md

```diff
@@ -0,0 +1,132 @@
+---
+keywords: [GreptimeDB, metadata storage, configuration, etcd, MySQL, PostgreSQL, metasrv, backend setup]
+description: Comprehensive guide for configuring metadata storage backends (etcd, MySQL, PostgreSQL) in GreptimeDB's metasrv component, including setup instructions and best practices.
+---
+
+# Configuration
+
+This section describes how to configure different metadata storage backends for the GreptimeDB metasrv component. The metadata storage is used to store critical system information including catalogs, schemas, tables, regions, and other metadata that are essential for the operation of GreptimeDB.
+
+## Available Storage Backends
+
+GreptimeDB supports the following metadata storage backends:
+
+- **etcd**: The default and recommended backend for development and testing environments
+- **MySQL**: Alternative backend for users who prefer MySQL
+- **PostgreSQL**: Alternative backend for users who prefer PostgreSQL
+
+## Use etcd as metadata storage
+
+:::warning
+etcd is only recommended for development and testing environments.
+:::
+
+### Configuration file
+
+Configure the metasrv component to use etcd as metadata storage:
+
+```toml
+# The metadata storage backend for metasrv
+backend = "etcd_store"
+
+# Store server addresses
+# You can specify multiple etcd endpoints for high availability
+store_addrs = ["127.0.0.1:2379"]
+```
+
+## Use MySQL as metadata storage
+
+:::tip
+For production environments, we strongly recommend using cloud providers' Relational Database Service (RDS) for metadata storage.
+:::
+
+MySQL can be used as a metadata storage backend. This is useful when you want to leverage existing MySQL infrastructure or have specific requirements for MySQL.
+
+
+### Configuration file
+
+Configure the metasrv component to use MySQL as metadata storage:
+
+:::warning
+If you want use same MySQL instance for metadata storage and application data, you need to configure the `meta_table_name` to avoid conflicts.
+:::
+
+```toml
+# The metadata storage backend for metasrv
+backend = "mysql_store"
+
+# Store server address
+# Format: mysql://user:password@ip:port/dbname
+store_addrs = ["mysql://user:password@ip:port/dbname"]
+
+# Optional: Custom table name for storing metadata
+# Default: greptime_metakv
+meta_table_name = "greptime_metakv"
+```
+
+## Use PostgreSQL as metadata storage
+
+:::tip
+For production environments, we strongly recommend using cloud providers' Relational Database Service (RDS) for metadata storage.
+:::
+
+PostgreSQL can be used as an alternative metadata storage backend. This is useful when you want to leverage existing PostgreSQL infrastructure or have specific requirements for PostgreSQL.
+
+### Configuration file
+
+Configure the metasrv component to use PostgreSQL as metadata storage:
+
+:::warning
+If you want use same PostgreSQL instance for metadata storage and application data, you need to configure the `meta_table_name` and `meta_election_lock_id` to avoid conflicts.
```

Comment:
Can you explain further what the conflict is? and how to choose the `meta_election_lock_id`?